verilog/tools/lint/lint_tool_test.sh

```diff
@@ -887,4 +887,60 @@ interactive_autofix_test "@^(y***D" \
 
 ################################################################################
 
+echo "=== Test --autofix=inplace-interactive: Choose alternative fix"
+
+# Choose alternatives with key '1' or '2'
+
+# Files with alternatives in autofixes
+ORIGINAL_ALT_AUTO_FIX="${TEST_TMPDIR}/orig-autofix-alternative.sv"
+>"${ORIGINAL_ALT_AUTO_FIX}"
+echo -en "module AlternativeAutoFix;\n" >>"${ORIGINAL_ALT_AUTO_FIX}"
```

Comment:
For multi-line echo sequences, I recommend the here-doc form:

```
cat > OUTPUT <<EOF
...
EOF
```

Reply:
Done (originally, I used the other way as it was commonly used in this file, mostly because the other examples were using various whitespace-relevant examples; but not needed in this case)